ydb/library/actors/wilson/wilson_uploader.cpp

```diff
@@ -136,28 +141,37 @@ namespace NWilson {
             void Bootstrap() {
                 Become(&TThis::StateWork);
 
+                if (MaxSpansPerSecond == 0) {
+                    ALOG_WARN(WILSON_SERVICE_ID, "max_spans_per_second should be greater than 0, changing to 1");
+                    MaxSpansPerSecond = 1;
+                }
+                if (MaxSpansInBatch == 0) {
+                    ALOG_WARN(WILSON_SERVICE_ID, "max_spans_in_batch shold be greater than 0, changing to 1");
+                    MaxSpansInBatch = 1;
+                }
+
                 TStringBuf scheme;
                 TStringBuf host;
                 ui16 port;
                 if (!TryGetSchemeHostAndPort(CollectorUrl, scheme, host, port)) {
-                    LOG_ERROR_S(*TlsActivationContext, WILSON_SERVICE_ID, "Failed to parse collector url (" << CollectorUrl << " was provided). Wilson wouldn't work");
+                    ALOG_ERROR(WILSON_SERVICE_ID, "Failed to parse collector url (" << CollectorUrl << " was provided). Wilson wouldn't work");
```

Comment:
We need some indicator here, health check light or a sensor. Do we have one?

Reply:
We don't at the moment, let's do it in a separate PR